src/Nri/Ui/Table/V2.elm

```diff
@@ -0,0 +1,281 @@
+module Nri.Ui.Table.V2
+    exposing
+        ( Column
+        , custom
+        , keyframeStyles
+        , keyframes
```

Comment:
> The keyframes styles are still exported separately. Can / should we include these with the other styles somehow?

I'm not sure if we can without wrapping the table and the style element in a div or something. [There doesn't seem to be a way of defining keyframes as an inline style](https://www.w3.org/TR/css-animations-1/#keyframes). Maybe a div wrapper isn't so bad?

I also feel like it can wait til [elm-css gains first-class animation support](https://github.com/rtfeldman/elm-css/issues/313)

Reply:
Makes sense! Going with the option to hold of on this until elm-css has first-class support.